source/get-started/create-a-connection-string.txt

```diff
@@ -0,0 +1,58 @@
+.. _ruby-get-started-connection-string:
+
+==========================
+Create a Connection String
+==========================
+
+You can connect to your MongoDB deployment by providing a
+**connection URI**, also called a *connection string*, which
+instructs the driver how to connect to a MongoDB deployment
+and how to behave while connected.
+
+The connection string includes the hostname or IP address and 
+port of your deployment, the authentication mechanism, user credentials 
+when applicable, and connection options.
+
+.. TODO:
+ To connect to an instance or deployment not hosted on Atlas, see
+ :ref:`ruby-connection-targets`.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Find your MongoDB Atlas connection string
+
+      To retrieve your connection string for the deployment that
+      you created in the :ref:`previous step <ruby-get-started-create-deployment>`,
+      log into your Atlas account and navigate to the
```

Comment:
[i] Per the [style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#l):
```suggestion
      log in to your Atlas account and navigate to the
```